lms/djangoapps/teams/tests/test_models.py

```diff
@@ -25,16 +25,18 @@ def setUp(self):
 
         self.user1 = UserFactory.create(username='user1')
         self.user2 = UserFactory.create(username='user2')
+        self.user3 = UserFactory.create(username='user3')
+
+        for user in (self.user1, self.user2, self.user3):
+            CourseEnrollmentFactory.create(user=user, course_id=COURSE_KEY1)
+        CourseEnrollmentFactory.create(user=self.user1, course_id=COURSE_KEY2)
 
         self.team1 = CourseTeamFactory(course_id=COURSE_KEY1, team_id='team1')
         self.team2 = CourseTeamFactory(course_id=COURSE_KEY2, team_id='team2')
 
-        self.team_membership11 = CourseTeamMembership(user=self.user1, team=self.team1)
```

Comment:
Correct. I changed these lines to call `add_user` just because it makes sense to test membership creation in the way that the actual application code does.